CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 [//]: # (current developments)
 
+## 24.11.0 (2024-11-15)
+
+### Enhancements
+
+* Introduce `--package-format` as a command line argument. (#4890 via #5209)
+  * This takes precedence over default value and `condarc`.
+  * Normalization occurs so `1`, `"1"`, `tar.bz2`,`.tar.bz2`, `2`, `"2"`, `conda`, `.conda` are all recognized and mapped appropriately.
+  * Other options are rejected.
+* Add  support for [CEP-17](https://github.com/conda/ceps/blob/main/cep-0017.md) that allows specifying the location of the site-packages directory with the `python_site_packages_path` build option for any packages named `python`. (#5502)
+
+### Bug fixes
+
+* Fix regex for Jinja2 `set` / `for` statements to be more specific. (#5514)
+* Fix `ruamel.yaml` usage to use supported APIs. (#5517)
+* Fix bug in docs build by pinning `conda-sphinx-theme` version to 0.2.2. (#5518)
```

Comment:
Does this belong to `# Docs` better?

Reply:
There is a bug in rever that miscounts things, was fixed in https://github.com/regro/rever/pull/266 but that was never released, so you likely used a version that miscounts contributions basically

Reply:
You can redo the run if you want with that new version